src/main/js/helpers/label.js

```diff
@@ -329,6 +329,7 @@ const createTooltipDiv = () => {
     d3.select("body")
         .append("div")
         .attr("id", styles.labelPopupTooltip)
+        .style("display", "none")
```

Comment:
Is it a passive change?

Reply:
No Veera, it will only fix the white box around graph issue. It won't effect anything other than that.